bucket/openscad-dev.json

```diff
@@ -1,50 +1,40 @@
 {
     "homepage": "https://www.openscad.org/",
-    "version": "2019.01-RC4",
+    "version": "2019.05.19",
     "license": "GPL-2.0-or-later",
     "description": "The Programmers Solid 3D CAD Modeller",
     "architecture": {
         "64bit": {
-            "url": "https://files.openscad.org/snapshots/OpenSCAD-2019.01-RC4-x86-64.zip",
-            "hash": "7279401226019cf387343f794f9bf4fc6ee679ef62351cbe6824e0d630f24679"
+            "url": "https://files.openscad.org/snapshots/OpenSCAD-2019.05.19.ci2704-x86-64.zip",
+            "hash": "7f1852cbdf9bb95d1f05c2ff43c7af0126cc64d3639f5a8dc40c7a838d8a8d81",
+            "extract_dir": "openscad-2019.05.19.ci2704"
         },
         "32bit": {
-            "url": "https://files.openscad.org/snapshots/OpenSCAD-2019.01-RC4-x86-32.zip",
-            "hash": "961d3f3f41a02d14192930566307d42024a9308b30ef77866a201de3584ca1a8"
+            "url": "https://files.openscad.org/snapshots/OpenSCAD-2019.05.19.ci2703-x86-32.zip",
+            "hash": "675d09f0e30347e716affaf9fe94185ad3114115b50b2ad1320784cb47e35097",
+            "extract_dir": "openscad-2019.05.19.ci2703"
         }
     },
-    "bin": [
-        [
-            "openscad.exe",
-            "openscad-dev"
-        ]
-    ],
-    "shortcuts": [
-        [
-            "openscad.exe",
-            "OpenSCAD Snapshot"
-        ]
-    ],
     "post_install": [
         "# OpenSCAD can't be started from a symlinked directory. See: https://github.com/openscad/openscad/issues/1309",
-        "startmenu_shortcut \"$original_dir\\openscad.exe\" \"OpenSCAD Snapshot\"",
-        "shim \"$original_dir\\openscad.exe\" $false \"openscad-dev\""
+        "startmenu_shortcut \"$original_dir/openscad.exe\" 'OpenSCAD Snapshot'",
+        "shim \"$original_dir/openscad.exe\" $false 'openscad-dev'"
```

Comment:
Why do you add inconsistency to manifests?

Almost every manifest and other codebase use windows backslashes for paths. Why to change it?

Reply:
I personally think forward-slashes are better, but I think consistency is more important. I will stop making this change unless it's decided to change it *everywhere* to forward-slash.